### DISCUSSION TITLE
Make long math expressions scrollable.

### DISCUSSION THREAD

**Discussion by dadezzz** [outline/outline#8032] [Ideas]
Just as we can scroll large code blocks horizontally, would it be possible to make math blocks scrollable if they are larger than the screen width?

**Comment by tommoor** [comment]
Could you provide an example of a math block that creates the situation? Thanks

**Reply by dadezzz** [reply]
Sure, this expression, for example, is wider than my phone's screen when it's rendered and it makes the whole page scroll horizontally:

```text
d(P, Q) = |\vec{PQ}| = \sqrt{(x_{Q} - x_{P})^{2} + (y_{Q} - y_{P})^{2} + (z_{Q} - z_{P})^{2}}
```

**Reply by tommoor** [reply]
In that case it definitely sounds like a bug

**Reply by dadezzz** [reply]
Ok, I thought it just wasn't implemented.

Probably it would be enough to set `overflow-x: scroll` in the css for the block containing the expression.

Should I open an issue?

**Reply by tommoor** [reply]
That particular isn't even close to being scrollable for me, even on mobile – but I believe it can happen :)

**Reply by dadezzz** [reply]
Yes, the expression is wider than the screen's width.

When this happens in code blocks they become scrollable horizontally but the rest of the page stays still.

With math blocks instead, the text from paragraphs around moves together when i scroll to the right to see the rest of the expression.

Here is an example:

![Screenshot_20241128-211948_Firefox.png](https://github.com/user-attachments/assets/355867e3-e50b-4895-b2a8-d227e2b502a0)

![Screenshot_20241128-211953_Firefox.png](https://github.com/user-attachments/assets/2c5e907d-148e-4317-8f66-39f08307e2ff)

If this is expected then I'll try to break expressions on multiple lines.

Thanks.

**Reply by tommoor** [reply]
Resolved in d66f41c854eaee054264eb00e420cc40c36a756c

**Reply by dadezzz** [reply]
Thank you!
